docs/how-to/namespace-transition.md

```diff
@@ -46,41 +75,84 @@ release from the `knative-gcp` repository. We will update the page with specific
    kubectl delete service controller -n cloud-run-events
    ```
 
-   For your convenience, we also packaged these instructions as a shell script
-   and as a yaml file that you can execute as shown below.
-
-   ```shell script
-   # execute the setup commands using the provided script
-   bash namespace-transition.sh
-   ```
+#### Using Workload Identity
 
-   Or, if you prefer executing these commands as a job, apply the following yaml
-   file.
+1. For this purpose we need to first setup a number of environment variables
+   based on the existing authentication settings. The following `gcloud` command
+   can be used to view the existing Google Service Accounts (GSAs) and their
+   roles in your GCP project.
 
    ```shell script
-   # execute the setup commands as a job
-   kubectl apply -f namespace-transition.yaml
+   export PROJECT_ID=<gcp-project-identifier>
+   # view GSAs and roles
+   gcloud projects get-iam-policy $PROJECT_ID
    ```
 
-   Once you confirm that the job has completed successfully, you can delete it.
+   Using this information, you can find the service accounts set up for the
+   control plane and broker data plane. The control plane GSA should have the
```

Comment:
These GSAs have default names, right? Most users will be using the defaults. Let's lead with those, and only proceed with looking at roles if the default names don't exist.

Also, we should give examples of the commands used to find the correct GSAs based on the list of roles. The user following these instructions may not be familiar with IAM commands. For example, I'm not familiar with IAM commands 😄

Reply:
I added a section for the default GSAs that were in the docs before the namespace change.

Unfortunately, I don't know how to do better than asking them to look for the correct GSAs by roles. The output of the `gcloud` command I already added is very verbose since it lists all the GSAs that exist in a project for each role. Using `grep` on the output is not particularly useful since the role is on a separate line at the bottom. In the future we will have a single role with a meaningful name and locating the GSAs will be a lot easier.